docs/api/resource_reaper.md

```
@@ -8,6 +8,49 @@ Testcontainers automatically assigns a Resource Reaper session id to each Docker
 
 Moby Ryuk derives its name from the anime character [Ryuk](https://en.wikipedia.org/wiki/Ryuk_(Death_Note)) and is a fitting choice due to the intriguing narrative of the anime Death Note.
 
+## Pinned Ryuk image
+
+Testcontainers for .NET pins the Ryuk image to this manifest list (OCI index) digest:
+
+```text
```

markdownlint-cli2 (0.21.0)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)